lms/envs/common.py

```diff
@@ -1213,9 +1213,9 @@ def _make_mako_template_dirs(settings):
 # Locale/Internationalization
 CELERY_TIMEZONE = 'UTC'
 TIME_ZONE = 'UTC'
-LANGUAGE_CODE = 'en'  # http://www.i18nguy.com/unicode/language-identifiers.html
+LANGUAGE_CODE = 'ar-sa'  # http://www.i18nguy.com/unicode/language-identifiers.html
 # these languages display right to left
-LANGUAGES_BIDI = ("he", "ar", "fa", "ur", "fa-ir", "rtl")
+LANGUAGES_BIDI = ("he", "ar", "ar-sa", "fa", "ur", "fa-ir", "rtl")
```

Comment:
@viadanna, @farhaanbukhsh, these can be overridden from `lms.yml`, right? If yes, can we drop these in favour of that method?

Reply:
I don't think so @lgp171188 , since YAML doesn't support tuple out of the box, and changing this through lms.yml seems force fitting. LANGUAGE_CODE is okay since that is meant to be altered through `lms.yml`. But other changes I feel we should keep in code.